aim/web/ui/src/services/api/api.ts

```diff
@@ -148,6 +145,17 @@ function remove<ResponseDataType>(url: string, options?: RequestInit) {
   });
 }
 
+function getTimezoneOffset(): string {
```

Comment:
do we really need this function, the body of function is too small and doesn’t reused it anywhere, IMO it would be better to remove it but if you want to reuse it later try to keep it in utils dir